templates/board_member/saved_profile.html.twig

```diff
@@ -3,24 +3,31 @@
 {% set currentRoute = app.request.attributes.get('_route') %}
 
 {% block content__inner %}
-    <h1>Saved profile page</h1>
-    <div id="saved_board_members_statistics">
-        <h1>Stats sur ces profils</h1>
-        <div>{{ stats.women }}% femmes / {{ stats.men }}% hommes</div>
-        <div>Moyenne d'âge : {{ stats.average_age }} ans</div>
-        <div>
-            {%- for area, stat in stats.areas -%}
-                {{ stat }}% {{ area|trans }}{% if not loop.last %} / {% endif %}
-            {%- endfor -%}
+    <section class="background--blue-light">
+        <div id="saved_board_members_statistics" class="l__wrapper space--60-0 spaces__search__results">
+            <h2 class="b__nudge--bottom">Stats sur ces profils</h2>
+            <ul class="text--body">
+                <li>{{ stats.women }}% femmes / {{ stats.men }}% hommes</li>
+                <li>{{ stats.average_age }} ans de moyenne d'âge</li>
```

Comment:
You need to change the test that checks this line.

Reply:
I changed it, thank you!!